nmtwizard/preprocess/prepoperator.py

```diff
@@ -263,6 +263,73 @@ def _postprocess_tu(self, tu):
         raise NotImplementedError()
 
 
+class MonolingualOperator(TUOperator):
+    """Base class for operations applying monolingual processing in each TU in a batch."""
+
+    def __init__(self, config, process_type, build_state):
+        self._postprocess_only = build_state.get("postprocess_only")
+        self._process_type = process_type
+        self._source_processor = None
```

Comment:
Can we find another name than processor? We already use `Processor` for something different so it can make things confusing.

Just `process` would be enough I think.